src/organisms/cards/RadioCard/index.js

```diff
@@ -41,16 +41,16 @@ function RadioCard(props) {
     children
   } = props;
 
+  const classes = [
+    styles['radio-card'],
+    input.value === radioValue && styles['checked'],
+    isValid(sections) && styles['sectioned'],
+    className
+  ];
+
   return (
     <div
-      className={
-        classnames(
-          styles['radio-card'],
-          { [styles.checked]: input.value === radioValue },
-          { [styles.sectioned]: isValid(sections) },
-          className
-        )
-      }
+      className={classnames(...classes)}
```

Comment:
Style preference really. No real upor downside to it other than I like using spread operator. Moving forward, we should use `const classes = classnames(...`